data/jobs.txt

```diff
@@ -1483,6 +1483,169 @@ mission "Core Pirate Attack"
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
 
+mission "Renegade Drones 1"
+	name "Neutralize broken Drone"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drone."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 20
+		"combat rating" > 200
+	clearance
```

Comment:
Hmm, I thought it would be weird if the player has a mission in favor of the planet but has to bribe it to get it's payment. On the other hand side this is definitely the player's problem and the other jobs don't seem to have it either. I'll remove the `clearance` tag.